### ISSUE TITLE
Incorrect answer passes test for Set the font-size for Multiple Heading Elements challenge

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/freeCodeCamp#43264]
I was assisting a camper on the forum and found out that incorrect answers are allowed to pass the test. 
This is my incorrect answer that still passes. 
```
<style>
<h1 style="font-size: 68px">This is h1 text</h1>
<h2 style="font-size: 52px">This is h2 text</h2>
<h3 style="font-size: 40px">This is h3 text</h3>
<h4 style="font-size: 32px">This is h4 text</h4>
<h5 style="font-size: 21px">This is h5 text</h5>
h6 {
  font-size:14px;
}
</style>
<h1 style="font-size: 68px">This is h1 text</h1>
<h2 style="font-size: 52px">This is h2 text</h2>
<h3 style="font-size: 40px">This is h3 text</h3>
<h4 style="font-size: 32px">This is h4 text</h4>
<h5 style="font-size: 21px">This is h5 text</h5>
<h6 style="font-size: 14px">This is h6 text</h6>
```
All of those headings in the style tags are supposed to be in the correct css format. 
But it looks like the test will just accept one of them in the correct format. 
## Affected page

https://www.freecodecamp.org/learn/responsive-web-design/applied-visual-design/set-the-font-size-for-multiple-heading-elements

## Correct code
This should be the only acceptable answer for this challenge. 

```
<style>
  h1 {
    font-size: 68px;
  }

  h2 {
    font-size: 52px
  }

  h3 {
    font-size: 40px;
  }

  h4 {
    font-size: 32px;
  }

  h5 {
    font-size: 21px;
  }

  h6 {
    font-size: 14px;
  }
</style>
<h1 style="font-size: 68px">This is h1 text</h1>
<h2 style="font-size: 52px">This is h2 text</h2>
<h3 style="font-size: 40px">This is h3 text</h3>
<h4 style="font-size: 32px">This is h4 text</h4>
<h5 style="font-size: 21px">This is h5 text</h5>
<h6 style="font-size: 14px">This is h6 text</h6>
```

## System

- Device: [e.g. iPhone6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.4]
- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]

**Comment by ieahleen** [comment]
The tests check for the computed css property
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/01-responsive-web-design/applied-visual-design/set-the-font-size-for-multiple-heading-elements.md

The easiest way would be to check that there is not inline style, adding a test maybe

**Comment by Nirajn2311** [comment]
Yes, we need another test to check if the user has used inline styles or not. Since jquery has been used check the heading font-size, I think we can go with a jquery solution for the new test.

We can use `.prop('style')` and check each of the heading if they have used inline style or not.

**Comment by ann-lang** [comment]
Hello, we are a group of researchers developing machine learning techniques to locate issues suitable for newcomers, and our model consider this issue as likely a "good first issue". May we recommend you to label it as "good first issue" so newcomers know where to choose?

Thank you!

**Comment by ieahleen** [comment]
Thanks for opening this issue.

This looks something that can be fixed by "first time" code contributors to this repository. Here are the files that you should be looking at to work on a fix:

List of files:

1. https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/01-responsive-web-design/applied-visual-design/set-the-font-size-for-multiple-heading-elements.md

The test text should be something like "You should not use inline styling".

Please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://chat.freecodecamp.org/channel/contributors) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

Sometimes we may get more than one pull requests. We typically accept the most quality contribution followed by the one that is made first.

Happy contributing.

**Comment by ShaunSHamilton** [comment]
# Addition

This is a prime candidate for the [CSSHelp](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/utils/css-help.ts) component. The tests could easily be changed to use it. It only looks for the styles within `style` tags, and will ignore those set by the inline style attribute.

There should be no need to check for inline styles - just change the tests to only check the values of the stylesheet.

_If you are unsure how to use the CSSHelp component, look in the adjacent `css-help.test.ts` file._

**Comment by srijit2002** [comment]
I am a beginner in open source. can someone please guide me where to start?

**Comment by ieahleen** [comment]
@srijit2002 start from reading my post above, it has what you need to start

**Comment by srijit2002** [comment]
@ieahleen can you please point me out which function of CSSHelp, I need to remove. So that, **_the tests only check the values of the stylesheet_**

**Comment by ieahleen** [comment]
@srijit2002 , none. The file you need to change is that of the challenge. You will use the CSSHelper to write the new tests

**Comment by ieahleen** [comment]
maybe using the CSSHelper is too complex for first timers

**Comment by srijit2002** [comment]
@ieahleen I am talking about the comment of @ShaunSHamilton 

>There should be no need to check for inline styles - just change the tests to only check the values of the stylesheet.

what are the changes I need to make?

**Comment by ShaunSHamilton** [comment]
@srijit2002 The tests currently check for the computed styles of the HTML elements. The `CSSHelp` component includes methods to parse the computed styles of the document stylesheet.

So, the `CSSHelp` component should be used instead of jQuery. The comment you quoted is saying there is not need to write a test looking for inline styles, because with the use of `CSSHelp` it does not matter if there are/aren't.

Hope this is clearer. Again, examples of using the `CSSHelp` component are in the adjacent `css-help.test.ts` file.

**Comment by srijit2002** [comment]
Yeah, now it is much clearer. Thanks for your precious time. I am working on this and link a pull request soon.
